eng/Publishing.props

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project>
+   <PropertyGroup>
+      <PublishingVersion>3</PublishingVersion>
```

Comment:
Could we add a comment about what this is for? If we know obviously.

Reply:
We can. For your ref: https://github.com/dotnet/arcade/blob/main/Documentation/CorePackages/Publishing.md#what-is-v3-publishing

Reply:
I really need to read arcade doc!